lib/rules/ngx-component-display.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
 
   create(context) {
     const options = context.options[0] || {};
-    const ignore = new RegExp(options.ignore || '^.*DialogComponent$');
+    const ignore = new RegExp(options.ignore || '^.*(?:Dialog|Modal)Component$');
```

Comment:
What is the purpose of "?:"? Why not simply "^.*(Dialog|Modal)Component$"?

Reply:
Almost similar: 

1. `^.*(?:Dialog|Modal)Component$` - This uses a non-capturing group `(?:...)`. It matches strings that end with "DialogComponent" or "ModalComponent", but it does not capture "Dialog" or "Modal" for later use.

2. `^.*(Dialog|Modal)Component$` - This uses a capturing group `(...)`. It also matches strings that end with "DialogComponent" or "ModalComponent", but it does capture "Dialog" or "Modal" for later use.

Since we don't need to use the matched "Dialog" or "Modal" later, i guess we can keep the first one, right?

Reply:
Or maybe you think that a detail and that will make some noise on the readme ?

Reply:
Indeed I think I think `^.*(Dialog|Modal)Component$` would be more clear but I am OK with your proposal.